### ISSUE TITLE
Collection view abruptly stops scrolling when dragging down keyboard

### ISSUE THREAD

**Issue by Jeyhey** [MessageKit/MessageKit#467]
## General Information

* MessageKit Version: 0.12.0

* iOS Version(s): 10

* Swift Version: 4

* Devices/Simulators: iPhone SE

* Reproducible in ChatExample? (Yes/No): Yes

##  What happened?
When dragging down the keyboard (quick drag down and release so that the collection view continues scrolling up) the collection view suddenly stops scrolling. This happens only if `maintainPositionOnKeyboardFrameChanged` is set to true.

The solution I found is the following. In `handleKeyboardDidChangeState(_ notification: Notification)` of `MessagesViewController` change
```
if maintainPositionOnKeyboardFrameChanged {
    messagesCollectionView.setContentOffset(contentOffset, animated: false)
}
```
to
```
if maintainPositionOnKeyboardFrameChanged && differenceOfBottomInset != 0 {
    messagesCollectionView.setContentOffset(contentOffset, animated: false)
}
```

This seems to fix the issue without having side effects on the behaviour of input bar and the collectionView.

## What did you expect to happen?
The collection view should continue scrolling, decelerate and finally slowly stop.

**Comment by zhongwuzw** [comment]
@Jeyhey , Thanks for your issue, you can adjust the code like below if you can make a `PR`.

```
let differenceOfBottomInset = afterBottomInset - messageCollectionViewBottomInset

if maintainPositionOnKeyboardFrameChanged && differenceOfBottomInset != 0 {
       let contentOffset = CGPoint(x: messagesCollectionView.contentOffset.x, y: messagesCollectionView.contentOffset.y + differenceOfBottomInset)
       messagesCollectionView.setContentOffset(contentOffset, animated: false)
}
```

**Comment by zhongwuzw** [comment]
This is resolved in 0.13.1. Thanks for your issue @Jeyhey .
